### PR TITLE
feat: add more test cases for my teams panel

### PR DESCRIPTION
**Pull request by kurokn**

### CODE REVIEW

File: app/jest/my-dashboard/my-teams/integrations.test.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    const viewIntegrationButton = screen.getByLabelText('view');
+    fireEvent.click(viewIntegrationButton);
+    expect(spyUseRouter).toHaveBeenCalled();
+  });
+
+  it('Should turn to correct page when click on the edit integration icon', async () => {
+    render(<MyTeams session={sampleSession} onLoginClick={jest.fn()} onLogoutClick={jest.fn()} key={'teams'} />);
+    fireEvent.click(await screen.findByRole('tab', { name: 'Integrations' }));
+
+    const viewIntegrationButton = screen.getAllByLabelText('edit');
```
Comment (NithinKuruba) [inline app/jest/my-dashboard/my-teams/integrations.test.tsx:126]: @kurokn please keep the naming right

**Comment by NithinKuruba** [comment]
@kurokn under `integrations` tab, the table has a status column and it shows the status of the integration. Please add a test to validate it. Different icon is displayed depending upon the integration status

File: app/jest/my-dashboard/my-teams/teamList.test.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    expect(screen.getByTitle('Delete team'));
+    await waitFor(() => {
+      fireEvent.click(screen.getByTestId('confirm-delete'));
+    });
+    expect(deleteTeam).toHaveBeenCalledTimes(1);
+  });
+
+  it('Should be able to click the Edit button, and save the new team name', async () => {
+    render(
+      <TeamList
```
Comment (NithinKuruba) [inline app/jest/my-dashboard/my-teams/teamList.test.tsx:128]: @kurokn please make sure to remove redundant code

File: app/jest/my-dashboard/my-teams/integrations.test.tsx (hunk 1 above)
Comment (kurokn) [reply app/jest/my-dashboard/my-teams/integrations.test.tsx]: renamed as `editIntegrationButton`

File: app/jest/my-dashboard/my-teams/teamList.test.tsx (hunk 2 above)
Comment (kurokn) [reply app/jest/my-dashboard/my-teams/teamList.test.tsx]: put the declarations into a helper.tsx

**Review by NithinKuruba — APPROVED** [review]

**Pull request merged**